ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/PreviewCard/W11_Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Windows_11.pdf",
```

Comment:
Is there a reason this is a .pdf file, rather than another file format?

Reply:
Generally we prefer having pdfs and having xcode generate the different sizes for us.
I think this one isn't quite set up correctly, but I'll have to look at another file to be sure. Less of an issue since it's just in our demo, though.
https://microsoft.github.io/apple-ux-guide/iOSImageFileFormat.html

Reply:
> Is there a reason this is a .pdf file, rather than another file format?

The SVG image would show as half of the image when exported. Noticed other images were uploaded in assets using the .pdf file format - attempted it and it worked without any issues . Should I retry SVG exporting?

Reply:
Actually do you think it could be a JPEG or PNG? The PDF is ~5.8 MB, but it's ultimately a raster (non-vector) image so one of those formats would probably be a lot smaller.